src/DynamoCoreWpf/ViewModels/Search/BrowserItemViewModel.cs

```diff
@@ -152,6 +153,22 @@ public string Assembly
             }
         }
 
+        public ElementTypeEnum ElementType
+        {
+            get
+            {
+                if (entries.All(entry => entry.ElementType == ElementTypeEnum.Package) &&
+                    subCategories.All(subCategory => subCategory.ElementType == ElementTypeEnum.Package))
+                    return ElementTypeEnum.Package;
+                else
+                    if (entries.All(entry => entry.ElementType == ElementTypeEnum.CustomDll) &&
+                        subCategories.All(subCategory => subCategory.ElementType == ElementTypeEnum.CustomDll))
+                        return ElementTypeEnum.CustomDll;
+                    else
+                        return ElementTypeEnum.RegularCategory;
+            }
+        }
```

Comment:
Hi, @Benglin . I thought about that and had little conversation with @elayabharath .
We can have next situation: in "Core" user adds some custom node or custom dll. But even with new added node/dll Core should remain as regular category... As well as other builtin categories e.g. operators, geometry.